app/views/admin/participants/_identities.html.erb

```diff
@@ -0,0 +1,25 @@
+<% @participant_identities.each_with_index do |identity, i| %>
+  <h3><%= "Identity #{i + 1} (#{identity_transferred_label(identity)})" %></h3>
+
+  <%= govuk_summary_list do |summary_list| %>
+    <% summary_list.row do |row| %>
+      <% row.key(text: "User ID / Participant ID" ) %>
+      <% row.value(text: tag.code(identity.user_id) ) %>
+    <% end %>
+
+    <% summary_list.row do |row| %>
+      <% row.key(text: "External ID" ) %>
+      <% row.value(text: tag.code(identity.external_identifier) ) %>
+    <% end %>
+
+    <% summary_list.row do |row| %>
+      <% row.key(text: "Email" ) %>
+      <% row.value(text: identity.email ) %>
+    <% end %>
+
+    <% summary_list.row do |row| %>
+      <% row.key(text: "Origin" ) %>
+      <% row.value(text: identity.origin ) %>
+    <% end %>
+  <% end %>
+<% end %>
```

Comment:
Could this use your new component?

Reply:
I moved the component outside the NPQ namespace to make it more generic and refactored the view to re use it.

Reply:
Nice!